doc/man3/SSL_shutdown.pod

```diff
@@ -35,35 +35,50 @@ performed, so that the peers stay synchronized.
 SSL_shutdown() supports both uni- and bidirectional shutdown by its 2 step
 behaviour.
 
-=over 4
+SSL_shutdown() only closes the communication in 1 direction.
```

Comment:
one could say the first SSL_shutdown closes only the write direction and
the second SSL_shutdown closes also the read direction.
It is a fatal condition when application data is received after the read direction is closed down.
That is very similar to tcp sockets, where that causes a RST.